app/models/yacht.rb

```diff
@@ -1,3 +1,12 @@
 class Yacht < ApplicationRecord
-  belongs_to :user
+  has_one_attached :photo
```

Comment:
I think this needs to be has_many_attached, no?

Reply:
sure